packages/graphql/tests/api-v6/integration/types/datetime.int.test.ts

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import neo4jDriver from "neo4j-driver";
+import type { UniqueType } from "../../../utils/graphql-types";
+import { TestHelper } from "../../../utils/tests-helper";
+
+describe("DateTime", () => {
+    const testHelper = new TestHelper({ v6Api: true });
+    let Movie: UniqueType;
+
+    beforeEach(() => {
+        Movie = testHelper.createUniqueType("Movie");
+    });
+
+    afterEach(async () => {
+        await testHelper.close();
+    });
+
+    describe("find", () => {
```

Comment:
I think that describe in describe is a bit messy, I'd rather have separate files here